Applications/MmrMobile/Contracts/AlertEvent.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace HomagConnect.MmrMobile.Contracts
+{
+    public class AlertEvent : IExtensibleDataObject
+    {
+        public AlertEvent(IEnumerable<KeyValuePair<string, string>> localizedSource, IEnumerable<KeyValuePair<string, string>> localizedMessage)
+        {
+            LocalizedSource = localizedSource;
+            LocalizedMessage = localizedMessage;
+        }
+
+        /// <summary>
+        /// Timestamp, when the Event started
+        /// </summary>
+        [JsonProperty(Order = 1)]
+        public DateTimeOffset StartTime { get; set; }
+
+        /// <summary>
+        /// Timestamp for the end of the event
+        /// If the event is still active, the current timestamp is returned
+        /// </summary>
+        [JsonProperty(Order = 2)]
+        public DateTimeOffset EndTime { get; set; }
+
+        /// <summary>
+        /// Unique identifier for one event
+        /// </summary>
+        [JsonProperty(Order = 2)]
+        public Guid InstanceId { get; set; }
+
+        /// <summary>
+        /// Machine-Number, Format "x-yyy-zz-nnnn"
+        /// </summary>
+        [JsonProperty(Order = 3)]
+        public string? MachineNumber { get; set; }
+
+        /// <summary>
+        /// value = 1 - 1000
```

Comment:
Please add the samples 600 / 800 as a comment so that it is possible to handle the ranges meaningful.